components/esp_websocket_client/esp_websocket_client.c

```diff
@@ -1228,7 +1228,7 @@ int esp_websocket_client_send_with_opcode(esp_websocket_client_handle_t client,
         goto unlock_and_return;
     }
     ret = esp_websocket_client_send_with_exact_opcode(client, opcode | WS_TRANSPORT_OPCODES_FIN, data, len, timeout);
-    if (ret <= 0) {
+    if (ret < 0) {
```

Comment:
Was this forgotten in ba335880088feb665d1beab29008203fbd91ca0e ?

Reply:
Yes, I rechecked it. From the return value perspective it is the same, in both case `less than 0` or `equal 0`, it will send the same return value. But to print `error message` should be only if it is less than 0, IMO.

Reply:
Also need to improve example tests, so it can check this changes. I will do it after.